package.json

```diff
@@ -0,0 +1,60 @@
+{
+	"name": "vl-ui-progress-bar",
+	"version": "0.0.0",
+	"description": "Gebruik de progress-bar component om de vooruitgang te laten zien van een proces dat uit verschillende stappen bestaat.",
+	"author": "DIDM",
+	"license": "MIT",
+	"main": "./src/index.js",
+	"exports": {
+		"require": "./index.js",
+		"import": "./src/index.js",
+		"node": "./index.js",
+		"default": "./index.js"
+	},
+	"homepage": "https://github.com/milieuinfo/webcomponent-vl-ui-progress-bar#readme",
+	"bugs": {
+		"url": "https://github.com/milieuinfo/webcomponent-vl-ui-progress-bar/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/milieuinfo/webcomponent-vl-ui-progress-bar.git"
+	},
+	"publishConfig": {
+		"registry": "http://artifactory-pr-build.lb.cumuli.be:8081/artifactory/api/npm/acd-npm/"
+	},
+	"scripts": {
+		"prepare": "npm update --dev vl-ui-util && npm run prepare:util",
+		"prepare:util": "npm explore vl-ui-util -- npm run util",
+		"test": "npm run test:unit && npm run test:e2e",
+		"test:unit": "wct --config-file ./wct.local.conf.json",
+		"test:e2e": "npm run test:server:start && npm run test:e2e:chrome && npm run test:e2e:firefox",
+		"test:e2e:grid": "npm run test:server:start && npm run test:e2e:chrome:grid && npm run test:e2e:firefox:grid",
+		"test:grid": "wct --config-file ./wct.docker.conf.json && npm run test:e2e:grid",
+		"posttest:e2e": "pkill -f http-server",
+		"test:e2e:chrome": "mocha --chrome",
+		"test:e2e:chrome:grid": "mocha --chrome --grid",
+		"test:e2e:firefox": "mocha --firefox",
+		"test:e2e:firefox:grid": "mocha --firefox --grid",
+		"test:e2e:opera": "mocha --opera",
+		"test:e2e:safari": "mocha --safari",
+		"test:server:start": "http-server &",
+		"release": "np",
+		"release:prepare": "npm explore vl-ui-util -- npm run release-component:prepare $(pwd) progress-bar",
+		"release:testless": "np --yolo",
+		"demo": "http-server"
+	},
+	"dependencies": {
+		"vl-ui-core": "^5.0.1",
+		"vl-ui-tooltip": "milieuinfo/webcomponent-vl-ui-tooltip#shadow_dom"
+	},
+	"peerDependencies": {
+		"vl-ui-core": "^5.0.1",
+		"vl-ui-tooltip": "milieuinfo/webcomponent-vl-ui-tooltip#shadow_dom"
+	},
+	"devDependencies": {
+		"@govflanders/vl-ui-util": "3.9.1",
+		"@govflanders/vl-ui-progress-bar": "3.9.1",
+		"vl-ui-util": "^5.0.7",
+		"vl-ui-button": "^5.0.0"
```

Comment:
indien niet nodig liever geen dependency

Reply:
ik dacht da we gingen resolven als je akkoord was met de opmerking? idem hierboven nog een paar ...

Reply:
Het is hier nodig, de kans dat een vl-ui-progress-bar dependency in de vl-ui-button komt zitten we met problemen en zal ik het persoonlijk na mijn uren rechttrekken.